srcpkgs/rpm/template

```diff
@@ -1,22 +1,22 @@
 # Template file for 'rpm'
 pkgname=rpm
-version=4.15.1
-revision=4
+version=4.16.0
+revision=2
```

Comment:
This should be `revision=1` when the version is new.

Reply:
the version has been fixed, thanks.